lib/rules/padding-line-between-tags.js

```diff
@@ -0,0 +1,217 @@
+      } else {
+        alwaysBlankLine.set(option.prev, [option.next])
+      }
+    } else {
+      const tagValue = neverBlankLine.get(option.prev)
+      if (tagValue) {
+        neverBlankLine.set(option.prev, [...tagValue, option.next])
+      } else {
+        neverBlankLine.set(option.prev, [option.next])
+      }
+    }
+  }
+
+  /**
+   * @param {VElement} block
+   */
+  return (block) => {
+    const endTag = block.endTag || block.startTag
+    const lowerSiblings = block.parent.children
+      .filter(
+        (element) =>
+          element.type === 'VElement' && element.range !== block.range
+      )
+      .filter((sibling) => sibling.range[0] - endTag.range[1] > 0)
+
+    if (lowerSiblings.length === 0) {
+      return
+    }
+
+    const closestSibling = lowerSiblings[0]
+
+    const element = /** @type {VElement} */ (closestSibling)
+    const lineDifference = element.loc.start.line - endTag.loc.end.line
+
+    if (
+      neverBlankLine.has('*') &&
+      // @ts-ignore
+      (neverBlankLine.get('*').includes('*') ||
+        // @ts-ignore
+        neverBlankLine.get('*').includes(element.name))
+    ) {
+      if (
+        alwaysBlankLine.has(block.name) &&
+        // @ts-ignore
+        (alwaysBlankLine.get(block.name).includes('*') ||
+          // @ts-ignore
+          alwaysBlankLine.get(block.name).includes(element.name))
+      ) {
+        if (lineDifference === 1) {
+          insertNewLine(context, block, element)
+        }
+      } else if (lineDifference > 1) {
+        removeExcessLines(context, endTag, element)
+      }
+    } else if (
+      alwaysBlankLine.has('*') &&
+      // @ts-ignore
+      (alwaysBlankLine.get('*').includes('*') ||
+        // @ts-ignore
+        alwaysBlankLine.get('*').includes(element.name))
+    ) {
+      if (
+        neverBlankLine.has(block.name) &&
+        // @ts-ignore
+        (neverBlankLine.get(block.name).includes('*') ||
+          // @ts-ignore
+          neverBlankLine.get(block.name).includes(element.name))
+      ) {
+        if (lineDifference > 1) {
+          removeExcessLines(context, endTag, element)
+        }
+      } else if (lineDifference === 1) {
+        insertNewLine(context, block, element)
+      }
+    } else {
+      if (
+        neverBlankLine.has(block.name) &&
+        // @ts-ignore
+        (neverBlankLine.get(block.name).includes('*') ||
+          // @ts-ignore
+          neverBlankLine.get(block.name).includes(element.name))
+      ) {
+        if (lineDifference > 1) {
+          removeExcessLines(context, endTag, element)
+        }
+      } else if (
+        alwaysBlankLine.has(block.name) &&
+        // @ts-ignore
+        (alwaysBlankLine.get(block.name).includes('*') ||
+          // @ts-ignore
+          alwaysBlankLine.get(block.name).includes(element.name)) &&
+        lineDifference === 1
+      ) {
+        insertNewLine(context, block, element)
+      }
+    }
+  }
+}
+
+module.exports = {
+  meta: {
+    type: 'problem',
```

Comment:
This should be also a layout rule, just like `padding-line-between-blocks`:

![grafik](https://user-images.githubusercontent.com/202916/189522259-d5216dec-da4a-4f0d-8d51-3306b040cd38.png)

(see https://deploy-preview-1966--eslint-plugin-vue.netlify.app/rules/)

```suggestion
    type: 'layout',
```